Gruntfile.js

```diff
@@ -46,59 +46,26 @@ module.exports = function(grunt) {
       }
     },
 
-    styleguide: {
-            dist: {
-                options: {
-                    name: 'eRegs Styleguide'
-                },
-                files: {
-                    '<%= env.frontEndPath %>/docs/styleguide': '<%= env.frontEndPath %>/css/'
-                }
-            }
-        },
-
     /**
-     * JSHint: https://github.com/gruntjs/grunt-contrib-jshint
+     * ESLint: https://github.com/sindresorhus/grunt-eslint
      *
-     * Validate files with JSHint.
+     * Validate files with ESLint.
      * Below are options that conform to idiomatic.js standards.
```

Comment:
This `Below...` line looks like it can be deleted, since the settings are no longer inline.